gui.js

```diff
@@ -1552,9 +1552,20 @@ IDE_Morph.prototype.fixLayout = function (situation) {
 IDE_Morph.prototype.setProjectName = function (string) {
     this.projectName = string.replace(/['"]/g, ''); // filter quotation marks
     this.hasChangedMedia = true;
+    this.updateDocumentTitleText();
```

Comment:
So, my hunch is, we can just write a single line with an assignment statement here, instead of introducing two additional functions that we don't ever call anywhere else...